sdks/python/apache_beam/runners/common.py

```diff
@@ -558,16 +568,33 @@ def __init__(self, window_fn, main_receivers, tagged_receivers):
     self.main_receivers = main_receivers
     self.tagged_receivers = tagged_receivers
 
+    # TODO: remove if block after per-element-output-counter released
+    # TODO: [BEAM-3937]
+    experiments = RuntimeValueProvider.get_value('experiments', str, [])
+    if 'outputs_per_element_counter' in experiments:
+      self.has_per_element_output_count = True
+      per_element_output_counter_name = \
```

Comment:
Please avoid \ line continuations.  Instead, do

```python
per_element_output_counter_name = (
    CounterName('per-element-output-count-meta', step_name=operation_name))
```

Reply:
will fix in next revision.